sdks/python/src/opik/message_processing/message_processors.py

```diff
@@ -63,6 +65,15 @@ def process(self, message: messages.BaseMessage) -> None:
             if exception.status_code == 409:
                 # sometimes a retry mechanism works in a way that it sends the same request 2 times.
                 # if the backend rejects the second request, we don't want users to see an error.
+                if isinstance(message, messages.AddThreadsFeedbackScoresBatchMessage):
```

Comment:
Let's maybe move it to `_process_add_threads_feedback_scores_batch_message`? It's quite specific for threads feedback scores.